bundles/org.openhab.core.test.magic/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -11,6 +11,17 @@
 		</config-description>
 	</thing-type>
 
+	<thing-type id="button">
+		<label>Magic Button</label>
+		<description>A button triggering different events.</description>
+
+		<channels>
+			<channel typeId="system.rawbutton" id="rawbutton"/>
+			<channel typeId="system.button" id="button"/>
+			<channel typeId="system.rawrocker" id="rawrocker"/>
```

Comment:
```suggestion
			<channel typeId="system.button" id="button"/>
			<channel typeId="system.rawrocker" id="rawrocker"/>
```

Reply:
Apparently we have all this different buttons.i the default system channel types:

https://github.com/openhab/openhab-core/blob/f9d201c35bacfe48270526419b8ac91bf488db30/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java#L67-L69

Reply:
It was still a bit early when I reviewed this so I thought it was using the same channel type twice. But I now see they are different. 🕵️‍♂️